plugin-tester-java/build.gradle

```diff
@@ -1,49 +1,26 @@
+buildscript {
+  repositories {
+    mavenLocal()
+  }
+
+  dependencies {
+    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.0.1'
```

Comment:
Maybe add a comment here explaining that this will be replaced with the 'link' to the currently-checked-out-version?